codes/quantum/qubits/surface/triangle.yml

```diff
@@ -0,0 +1,56 @@
+    - 'For decoding a single layer of the triangle code, a decoding graph is laid on the code. Nodes of the graph are located at each stabilizer (center of the triangle graph) and has red or blue edges where red associates with \(X\) errors and blue with \(Z\) errors. To take into account any errors from measuring the error syndrome, a three-dimensional stack of the decoding graphs is laid ontop of the code with vertical edges connecting to qubits between layers \cite{doi:10.22331/q-2017-04-25-2}.'
+
+  fault_tolerance:
+    - 'The symmetry of triangle codes allows for fault-tolerant measurement and initialization in any Pauli basis \cite{doi:10.22331/q-2017-04-25-2}.'
+    - 'A non-fault-tolerant curcuit initializes the triangle code then to guarantee fault-tolerance, this is followed by postselection on trivial measurements of the syndrome and of the appropriate logical Pauli based on the basis of the logical states \cite{doi:10.22331/q-2017-04-25-2}.'
+    - 'For syndrome extraction, instead of using the conventional ordering of coupling to the loop after coupling to the plaquette, the extraction has the coupling to the loop interwoven with the coupling to the plaquette. These are equivalent in circuits, but makes a different to the fault-tolerance since the latter can detect hook errors and the former cannot \cite{doi:10.22331/q-2017-04-25-2}.'
+
+  code_capacity_threshold:
+    - '\(10\%\) bit-flip error-correction threshold for ideal syndrome measurements and \(10\%\) bit-phase flip error-correction threshold for ideal syndrom measurements \cite{doi:10.22331/q-2017-04-25-2}.'
+    - '\(3.2\%\) bit-flip error-correction threshold for noisy syndrome measurements and \(2.6\%\) for bit-phase flip noise \cite{doi:10.22331/q-2017-04-25-2}.'
+
+  threshold:
+    - 'For a triangle code with 7 qubits of information, 6 ancilla qubits, 13 qubits in total, and max degree of 4 the lower pseudothreshold is \(1.05 \times 10^{-4}\) and upper pseudothreshold is \(1.22 \times 10^{-4}\) \cite{doi:10.22331/q-2017-04-25-2}.'
+    - 'For a triangle code with 7 qubits of information, 6 ancilla qubits, 13 qubits in total, and max degree of 5 the lower pseudothreshold is \(1.57 \times 10^{-4}\) and upper pseudothreshold is \(1.92 \times 10^{-4}\) \cite{doi:10.22331/q-2017-04-25-2}.'
+    - 'For a triangle code with 7 qubits of information, 8 ancilla qubits, 15 qubits in total, and max degree of 4 the lower pseudothreshold is \(1.76 \times 10^{-4}\) and upper pseudothreshold is \(2.23 \times 10^{-4}\) \cite{doi:10.22331/q-2017-04-25-2}.'
+
+realizations:
+  - 'Code used to improve surface codes by creating a way to implement gate \(S\) without state distillation \cite{doi:10.22331/q-2017-04-25-2}.'
+
+notes:
+  - 'Several tables and helpful figured to more easily see what the code looks like can be found in \cite{doi:10.22331/q-2017-04-25-2}.'
+
+relations:
+  parents:
+    - code_id: surface #not sure on this since these are CSS codes, but triangle are non-CSS
```

Comment:
change parent to clifford-deformed_surface, and mention that these are non-CSS derivatives of the surface code.